From dfd50cf965643f830cf18c6d3d3dd9b5fd9c224a Mon Sep 17 00:00:00 2001
From: Don Zickus <dzickus@redhat.com>
Date: Tue, 16 Sep 2014 17:02:32 -0400
Subject: [usb] core/devio: Spaces to tabs for proc_reapurbnonblock()

Message-id: <1410887463-20674-81-git-send-email-dzickus@redhat.com>
Patchwork-id: 95409
O-Subject: [RHEL7 PATCH 080/591] drivers: usb: core: devio.c: Spaces to tabs for proc_reapurbnonblock()
Bugzilla: 1110939
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

https://bugzilla.redhat.com/show_bug.cgi?id=1110939

commit 5b32c385bef17fd3d3b8ce7d5af2ee2aa2225ff5
Author: Matthias Beyer <mail@beyermatthias.de>
Date:   Mon Oct 14 21:46:36 2013 +0200

    drivers: usb: core: devio.c: Spaces to tabs for proc_reapurbnonblock()

    Replaced spaces by tabs for proc_reapurbnonblock() function.

    Signed-off-by: Matthias Beyer <mail@beyermatthias.de>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/usb/core/devio.c b/drivers/usb/core/devio.c
index bd429ea..c697463 100644
--- a/drivers/usb/core/devio.c
+++ b/drivers/usb/core/devio.c
@@ -1634,14 +1634,14 @@ static int proc_reapurbnonblock(struct dev_state *ps, void __user *arg)
 static int proc_control_compat(struct dev_state *ps,
     struct usbdevfs_ctrltransfer32 __user *p32)
 {
-        struct usbdevfs_ctrltransfer __user *p;
-        __u32 udata;
-        p = compat_alloc_user_space(sizeof(*p));
-        if (copy_in_user(p, p32, (sizeof(*p32) - sizeof(compat_caddr_t))) ||
-            get_user(udata, &p32->data) ||
+ struct usbdevfs_ctrltransfer __user *p;
+ __u32 udata;
+ p = compat_alloc_user_space(sizeof(*p));
+ if (copy_in_user(p, p32, (sizeof(*p32) - sizeof(compat_caddr_t))) ||
+     get_user(udata, &p32->data) ||
      put_user(compat_ptr(udata), &p->data))
   return -EFAULT;
-        return proc_control(ps, p);
+ return proc_control(ps, p);
 }
 
 static int proc_bulk_compat(struct dev_state *ps,
-- 
1.7.1